src/SQLStore/ListLookup/UndeclaredPropertyListLookup.php

```diff
@@ -99,20 +99,32 @@ public function getLookupIdentifier() {
 	private function selectPropertiesFromTable( $propertyTable ) {
 
 		$options = $this->store->getSQLOptions( $this->requestOptions, 'title' );
+		$idTable = $this->store->getObjectIds()->getIdTable();
+
 		$options['ORDER BY'] = 'count DESC';
+		$options['GROUP BY'] = 'smw_title';
+
+		$conditions = array(
+			'smw_id > 50', // FIXME Use \SMWSql3SmwIds::PPBORDERID
+			'page_id IS NULL',
+		);
 
 		$db = $this->store->getConnection( 'mw.db' );
 
-		// TODO: this is not how JOINS should be specified in the select function
 		$res = $db->select(
-			$propertyTable->getName() . ' INNER JOIN ' .
-			$db->tablename( $this->store->getObjectIds()->getIdTable() ) . ' ON p_id=smw_id LEFT JOIN ' .
```

Comment:
There was either a way to escape all tables with `$db->tablename` or to use proper MW/join syntax and let the `select` handle the DB prefix.